dashboard/modules/dashboard_sdk.py

```diff
@@ -125,8 +126,16 @@ def parse_cluster_info(
     headers: Optional[Dict[str, Any]] = None,
 ) -> ClusterInfo:
     if address is None:
-        logger.info(f"No address provided, defaulting to {DEFAULT_DASHBOARD_ADDRESS}.")
-        address = DEFAULT_DASHBOARD_ADDRESS
+        if ray.is_initialized():
```

Comment:
Sure– what does this new address point to:

```python
address = f"http://{ray._private.worker.global_worker.node.address_info['webui_url']}"
```

What's the motivation for changing it from the dashboard address? We currently default to that because Job Submission and the Serve CLI need to access the Ray dashboard and they use this SDK to do so.